### ISSUE TITLE
Wrong generation of List

### ISSUE THREAD

**Issue by fpotera** [stoicflame/enunciate#582]
Hello.
We generate api for the following class:

```java
  import java.util.ArrayList;
  import java.util.List;

  public class MapType<MapEntry>
  {
	private List<MapEntry> entries = new ArrayList<>();

	public void addEntry(MapEntry entry)
	{
		entries.add(entry);
	}

	public List<MapEntry> getEntries()
	{
		return entries;
	}
  }
```

It's generated in swagger.json as:

```json
    "json_MapType" : {
      "type" : "object",
      "title" : "MapType",
          "properties" : {
            "entries" : {
"description" : "",
"type" : "array",
"items" : {
"type" : "array",
"items" : {
  "type" : "file"
}
}
            }
          },
      "description" : ""
    }
```

I think "file" type is not right and it should be a "ref" to other object.

**Comment by stoicflame** [comment]
Fixed in 2.9.

**Comment by stoicflame** [comment]
[Enunciate 2.9.0 has been released.](https://github.com/stoicflame/enunciate/releases/tag/v2.9.0)
